Core/Utilities/TimeLeft/SGETimeLeft.py

```diff
@@ -43,39 +43,6 @@ def getResourceUsage( self ):
     result = runCommand( cmd )
     if not result['OK']:
       return result
-    _example = """ Example of output from qstat -f -j $JOB_ID
```

Comment:
I understand that this return string has been put in the test, but was it necessary to remove it from here as it is a good "documentation" of what is expected and why the code is how it is (i.e. messy!)